deepchecks/utils/distribution/drift.py

```diff
@@ -24,9 +24,45 @@
     preprocess_2_cat_cols_to_same_bins
 from deepchecks.utils.strings import format_percent
 
+__all__ = ['calc_drift_and_plot']
+
+
 PSI_MIN_PERCENTAGE = 0.01
+SUPPORTED_CATEGORICAL_METHODS = ['Cramer', 'PSI']
+SUPPORTED_NUMERICAL_METHODS = ['Earth Mover\'s Distance']
 
-__all__ = ['calc_drift_and_plot']
+
+def cramers_v(dist1: Union[np.ndarray, pd.Series], dist2: Union[np.ndarray, pd.Series]) -> float:
+    """Calculate the Cramer's V statistic.
+
+    For more on Cramer's V, see https://en.wikipedia.org/wiki/Cram%C3%A9r%27s_V
+    Uses the Cramer's V bias correction, see http://stats.lse.ac.uk/bergsma/pdf/cramerV3.pdf
+
+    Function is for categorical data only.
+
+    Parameters
+    ----------
+    dist1 : Union[np.ndarray, pd.Series]
+        array of numberical values.
+    dist2 : Union[np.ndarray, pd.Series]
+        array of numberical values to compare dist1 to.
+    Returns
+    -------
+    float
+        the bias-corrected Cramer's V value of the 2 distributions.
+
+    """
+    dist1_counts, dist2_counts, _ = preprocess_2_cat_cols_to_same_bins(dist1=dist1, dist2=dist2)
+    contingency_matrix = pd.DataFrame([dist1_counts, dist2_counts])
+
+    chi2 = chi2_contingency(contingency_matrix)[0]
+    n = contingency_matrix.sum().sum()
+    phi2 = chi2/n
+    r, k = contingency_matrix.shape
+    phi2corr = max(0, phi2 - ((k-1)*(r-1))/(n-1))
+    rcorr = r - ((r-1)**2)/(n-1)
+    kcorr = k - ((k-1)**2)/(n-1)
+    return np.sqrt(phi2corr / min((kcorr-1), (rcorr-1)))
```

Comment:
This is based on https://stackoverflow.com/questions/46498455/categorical-features-correlation/46498792#46498792  and is reused in other sources ( https://towardsdatascience.com/the-search-for-categorical-correlation-a1cf7f1888c9 ) and I think we need to give credit. @noamzbr ?

Reply:
added for now